.github/workflows/task-lint-cargo.yml

```
@@ -45,3 +45,13 @@ jobs:
           echo "Running cargo clippy..."
           cargo clippy --workspace --no-deps -- -D warnings
           cargo clippy --workspace --tests --no-deps -- -D warnings
+
+          # TODO(mehul, hotfix): This is a temporary fix to ensure that the madara is linted.
```

I think this practice that's being followed by starkware team!
```suggestion
          # TODO(mehul 14/11/2025, hotfix): This is a temporary fix to ensure that the madara is linted.
```